flake.nix

```diff
@@ -154,6 +154,13 @@
     processComposeModules.default =
       import ./nix/external-services.nix { inherit inputs; };
     environments.externalServices = import ./nix/external-services-envs.nix;
+
+    om.ci.default = {
+      simple-example = {
+        overrideInputs = { common = ./.; };
```

Comment:
I'm pretty sure this shuld be `common = "."`. 

(But the point is mute when using `om.yaml` anyway)